.travis.yml

```diff
@@ -25,6 +25,8 @@ matrix:
     # Meta
     - python: "3.6"
       env: TOXENV=flake8
+    - python: "3.6"
+      env: TOXENV=isort-check
```

Comment:
Is it worth a single job?
There could be one `checkqa` job.
And since there is flake8 already, what about using https://github.com/gforcada/flake8-isort ?

Reply:
Excellent point, fixed.